libs/platform/src/lib/components/button/button.component.ts

```diff
@@ -11,13 +11,25 @@ export type ButtonType =
     | 'transparent'
     | 'emphasized'
     | 'menu';
+
+export type GlyphPosition = 'before' | 'after';
```

Comment:
We should not create our own type here. Just use the one created in core.

https://github.com/SAP/fundamental-ngx/blob/3fff22203122155c4308c093f7956734a23bfa52/libs/core/src/lib/button/button.component.ts#L13

Reply:
removed